spanner/api/Spanner.Samples/InsertUsingDmlReturningAsync.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_insert_dml_returning]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+public class InsertUsingDmlReturningAsyncSample
+{
+    public async Task<List<string>> InsertUsingDmlReturningAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        var insertedSingers = new List<string>();
+        using var connection = new SpannerConnection(connectionString);
+        await connection.OpenAsync();
+
+        // Insert records into the SINGERS table and returns the
+        // generated column FullName of the inserted records using
+        // ‘THEN RETURN FullName’.
+        // It is also possible to return all columns of all the
+        // inserted records by using ‘THEN RETURN *’.
+        using var cmd = connection.CreateDmlCommand(@"INSERT INTO Singers(SingerId, FirstName, LastName) VALUES
```

Comment:
The indentation seems off here. Could you please check the existing samples and correct/respond?

Reply:
Done.